core/chain_util.go

```diff
@@ -52,8 +53,20 @@ func CalcDifficulty(time, parentTime uint64, parentDiff *big.Int) *big.Int {
 		diff.Sub(parentDiff, adjust)
 	}
 	if diff.Cmp(params.MinimumDifficulty) < 0 {
-		return params.MinimumDifficulty
+		diff = params.MinimumDifficulty
 	}
+
+	periodCount := new(big.Int).Add(number, big.NewInt(1))
```

Comment:
On that specific line it would be he block number. But if you look one line below that one you'll see it's a period :)